tasks/maketextfiles.js

```diff
@@ -20,17 +20,24 @@ module.exports = function (grunt) {
         var path = require('path');
         var _ = require('lodash');
 
-        //static vars
         var packageFileName = 'package.json';
-        var projectBasePath = 'project/';
-        var projectPackage = grunt.file.readJSON(packageFileName);
-        //template file relative to this file
-        var templateFileName = '../templates/textFiles.template.js';
 
         // Merge task-specific and/or target-specific options with these defaults.
         var options = this.options({
+            projectBasePath: 'project/',
             destinationFileName: projectBasePath + 'textFiles.js'
```

Comment:
Woops! Bug here: `projectBasePath` is undefined at this point. Fixed and I've released as [`v1.1.0`](https://github.com/incuna/grunt-maketextfiles/releases/tag/v1.1.0)